build.sbt

```diff
@@ -10,7 +10,7 @@ lazy val commonSettings = Seq(
   organizationHomepage := Some(url("http://www.indix.com")),
   scalaVersion := "2.11.11",
   scalacOptions ++= Seq("-encoding", "UTF-8", "-deprecation", "-unchecked"),
-  javacOptions ++= Seq("-Xlint:deprecation", "-source", "1.7"),
+  javacOptions ++= Seq("-Xlint:deprecation", "-source", "1.8"),
```

Comment:
Do we need to enforce 1.8? Scala 2.11 can work with 1.6. Forcing this to 1.8 might break compatibility with Scala 2.11 assumption. Unless we're using Lamdas' and Streams API introduced in JDK8.

Reply:
We are using lambdas in the gocd-util module. So, I put that in common settings. Should I can make it module specific only?

Reply:
Ah Okay, see the above message on moving to Scala. If we decide to do that this comment becomes redundant.